core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java

```diff
@@ -91,8 +91,15 @@ public void onResponse(IndicesStatsResponse indicesStatsResponse) {
                         IndexShardStats shard = indicesStatsResponse.getIndex(sourceIndex).getIndexShards().get(i);
                         return shard == null ? null : shard.getPrimary().getDocs();
                     }, indexNameExpressionResolver);
-                createIndexService.createIndex(updateRequest, ActionListener.wrap(response ->
-                    listener.onResponse(new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked())), listener::onFailure));
+                createIndexService.createIndex(
+                    updateRequest,
+                    ActionListener.wrap(response ->
+                        listener.onResponse(
+                            new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked(), updateRequest.index())
+                        ),
```

Comment:
Sorry to nit on this, I hate formatting nits too but we still have to keep the codebase clean, but I think it should fit like this:

```diff
diff --git a/core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java b/core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java
index 9c6855c1a5..2555299709 100644
--- a/core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java
+++ b/core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java
@@ -94,9 +94,7 @@ public class TransportShrinkAction extends TransportMasterNodeAction<ShrinkReque
                 createIndexService.createIndex(
                     updateRequest,
                     ActionListener.wrap(response ->
-                        listener.onResponse(
-                            new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked(), updateRequest.index())
-                        ),
+                        listener.onResponse(new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked(), updateRequest.index())),
                         listener::onFailure
                     )
                 );
```